guide/07-working-with-bigdata/02-Making-your-data-accessible-to-GA-server.ipynb

```diff
@@ -0,0 +1,730 @@
+{
```

Comment:
<p>..</p><p>HDFS - <strong>a</strong>&nbsp;<a href="https://hadoop.apache.org/docs/r1.2.1/hdfs_design.html#Introduction" target="_blank">Hadoop Distributed File System</a>&nbsp;directory of datasets</p><p>..</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/804/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>